cmd/blockchaincmd/create.go

```diff
@@ -367,7 +372,55 @@ func createBlockchainConfig(cmd *cobra.Command, args []string) error {
 		); err != nil {
 			return err
 		}
-	} else {
+	case models.HyperVM:
+		if genesisPath == "" {
+			providePath := "I'll provide the genesis path"
+			binaryGen := "The VM binary will generate the genesis"
+			defaultGen := "Use the DefaultGenesis for my VM"
+			option, err := app.Prompt.CaptureList(
+				"How would you like to provide the genesis file?",
+				[]string{providePath, binaryGen, defaultGen},
+			)
+			if err != nil {
+				return err
+			}
+			switch option {
+			case providePath:
```

Comment:
no need to add another prompt to ask for path. better UX is to just ask user to provide path to genesis right away and use it after checking if file exists. So this case branch is not needed here  and can be moved  up to `How would you like to provide the genesis file?` prompt